eva-accession-import/src/test/java/uk/ac/ebi/eva/accession/dbsnp/io/SubSnpNoHgvsContigAndChromosomeReaderTest.java

```diff
@@ -60,37 +61,37 @@ public void tearDown() {
     @Test
     public void readChickenVariants() throws Exception {
         reader = buildReader(CHICKEN_ASSEMBLY_5, PAGE_SIZE);
-        List<String> variants = readAll(reader);
-        assertEquals(6, variants.size());
+        List<CoordinatesPresence> coordinates = readAll(reader);
+        assertEquals(7, coordinates.size());
```

Comment:
Maybe we should we check some of the values in these, e.g. to ensure the chromosome/contig are correctly assigned, and that the boolean flag is set.